Advanced Select/Type of Triangle.sql

```diff
@@ -5,5 +5,5 @@ when A + B <= C or A + C <= B or B + C <= A then "Not A Triangle"
 when A = B and B = C then "Equilateral"
 when A = B or A = C or B = C then "Isosceles"
 else "Scalene"
-end as triangle_sides
-from TRIANGLES
+end as triangle_sides 
+
```

Comment:
Could you please delete this file and pull it again since there's no change in the file. This way it will not show a change in the PR. Thanks!